src/components/edit-drop-down-lists/edit-drop-down-lists-page.jsx

```diff
@@ -18,7 +18,7 @@ const EditDropDownListsPage = () => {
     fetch(`${endpointUrl}/print-inputs-options`)
```

Comment:
We need credentials: "include" here as well - the BE needs to check if jwt cookie is authorised before returning data.